packages/fiori/src/Wizard.ts

```diff
@@ -233,6 +234,13 @@ type StepInfo = {
 })
 
 class Wizard extends UI5Element {
+	/**
+	 * Defines how the content of the <code>ui5-wizard</code> would be visualized.
```

Comment:
Many tags are missing - type, since and defaultValue. You can check other public properties for reference

Reply:
Done.